old-github-actions/build_wheels.old

```diff
@@ -2,7 +2,9 @@ name: Build wheels
 
 on:
   # Trigger on pull requests.
-  pull_request:
+  # commenting this out now so we can focus on new tests
+  # TODO remove this file before merging
+  #pull_request:
```

Comment:
You don't need to uncomment this, you could remove it if you are concerned about the duplicate work or extra time to use the new `run_tests` and cibuildwheel. This way, cibuildwheel to only test on `master` after PRs are merged.

You need to keep cibuildwheel as it builds and uploads the wheels to pypi.

Reply:
I'll move it back to where it was. I will also uncomment for now so I can make sure it works with the new format, but I don't want it to trigger on PRs when this is PR is eventually merged.